app/views/devise/confirmations/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
+
```

Comment:
Since we're not adding the theme styles to this view yet or overriding its functionality, we can remove it for now